packages/api/src/controllers/stripe.ts

```diff
@@ -214,6 +225,70 @@ const sendUsageRecordToStripe = async (
   );
 };
 
+export const calculateOverageOnMinimumSpend = async (product, billingUsage) => {
+  const minimumSpend = product.monthlyPrice;
+
+  let storageUsage = billingUsage.storageUsage;
+  let deliveryUsage = billingUsage.deliveryUsage;
+  let totalUsage = billingUsage.totalUsage;
+
+  let prices: any = {};
+
+  if (product?.usage) {
+    product.usage.forEach((item) => {
+      if (item.name.toLowerCase() === "transcoding")
+        prices.transcoding = item.price;
+      if (item.name.toLowerCase() === "delivery") prices.streaming = item.price;
+      if (item.name.toLowerCase() === "storage") prices.storage = item.price;
```

Comment:
**nit:** Please avoid blocks of code without curly braces as those are pretty error-prone as the code evolves, especially if multi-line. 
```suggestion
      if (item.name.toLowerCase() === "transcoding") {
        prices.transcoding = item.price;
      } else if (item.name.toLowerCase() === "delivery") {
        prices.streaming = item.price;
      } else if (item.name.toLowerCase() === "storage") {
        prices.storage = item.price;
      }
```

(it's also easier to read IMO, with less stuff happening on each line)